apps/src/storage/dataBrowser/DataVisualizer.jsx

```diff
@@ -115,14 +81,7 @@ class DataVisualizer extends React.Component {
         }
         break;
       case 'Cross Tab':
-        if (this.state.xValues && this.state.yValues) {
-          chart = new GoogleChart.CrossTab(targetDiv);
-          ({chartData, columns} = this.createPivotTable(
-            this.state.parsedRecords,
-            this.state.xValues,
-            this.state.yValues
-          ));
-        }
+        // No GoogleChart for Cross Tab
```

Comment:
It might be more helpful for this comment to add a positive assertion about the Cross Tab chart, like "Cross Tab is rendered by React, so no Google Chart is required."